src/corelibs/U2View/src/ov_sequence/AnnotationsTreeView.cpp

```diff
@@ -674,51 +674,51 @@ void AnnotationsTreeView::sl_onAnnotationsRemoved(const QList<Annotation *> &as)
         switch (annotationModification.type) {
-        case AnnotationModification_NameChanged:
-        case AnnotationModification_LocationChanged:
-        case AnnotationModification_TypeChanged: {
-            QList<AVAnnotationItem *> aItems = findAnnotationItems(annotationModification.annotation);
-            assert(!aItems.isEmpty());
-            foreach (AVAnnotationItem *ai, aItems) {
-                ai->updateVisual(ATVAnnUpdateFlag_BaseColumns);
-            }
-        } break;
+            case AnnotationModification_NameChanged:
+            case AnnotationModification_LocationChanged:
+            case AnnotationModification_TypeChanged: {
+                QList<AVAnnotationItem *> aItems = findAnnotationItems(annotationModification.annotation);
+                assert(!aItems.isEmpty());
+                foreach (AVAnnotationItem *ai, aItems) {
+                    ai->updateVisual(ATVAnnUpdateFlag_BaseColumns);
+                }
+            } break;
 
-        case AnnotationModification_QualifierRemoved: {
-            const QualifierModification &qm = static_cast<const QualifierModification &>(annotationModification);
-            QList<AVAnnotationItem *> aItems = findAnnotationItems(qm.annotation);
-            foreach (AVAnnotationItem *ai, aItems) {
-                ai->removeQualifier(qm.getQualifier());
-            }
-        } break;
-        case AnnotationModification_QualifierAdded: {
-            const QualifierModification &qm = static_cast<const QualifierModification &>(annotationModification);
-            QList<AVAnnotationItem *> aItems = findAnnotationItems(qm.annotation);
-            foreach (AVAnnotationItem *ai, aItems) {
-                if (ai->isExpanded() || ai->childCount() > 1) {    //if item was expanded - add real qualifier items
-                    ai->addQualifier(qm.getQualifier());
-                } else {
-                    ai->setChildIndicatorPolicy(QTreeWidgetItem::ShowIndicator);    //otherwise process indicator only
+            case AnnotationModification_QualifierRemoved: {
+                const QualifierModification &qm = static_cast<const QualifierModification &>(annotationModification);
+                QList<AVAnnotationItem *> aItems = findAnnotationItems(qm.annotation);
+                foreach (AVAnnotationItem *ai, aItems) {
+                    ai->removeQualifier(qm.getQualifier());
                 }
-            }
-        } break;
-        case AnnotationModification_AddedToGroup: {
-            const AnnotationGroupModification &gmd = static_cast<const AnnotationGroupModification &>(annotationModification);
-            AVGroupItem *gi = findGroupItem(gmd.getGroup());
-            SAFE_POINT(NULL != gi, L10N::nullPointerError("annotation view group item"), );
-            buildAnnotationTree(gi, gmd.annotation);
-            gi->updateVisual();
-        } break;
+            } break;
```

Comment:
Newline missed

Reply:
This is how our clangformat style is set up. I also do not like this style - we need to update clangformat and reformat all sources. Will do it in the separate PR next.